package.json

```diff
@@ -25,9 +25,10 @@
     "gulp-mocha": "^2.2.0",
     "gulp-rename": "^1.2.2",
     "gulp-size": "^2.0.0",
-    "gulp-uglify": "^1.4.2",
+    "gulp-uglify": "^1.5.4",
```

Comment:
Per [the guidance of the README](https://github.com/terinjokes/gulp-uglify#using-a-different-uglifyjs), this is a valid configuration for minifying ES6 with `gulp-uglify`. I did not investigate other packages, although I suspect there is more than 30 ways to skin this cat.